app/fissile.go

```diff
@@ -39,6 +40,26 @@ func NewFissileApplication(version string, ui *termui.UI) *Fissile {
 	}
 }
 
+// SetPatchPropertiesDirective saves the patch-properties release and job names, if specified.
+func (f *Fissile) SetPatchPropertiesDirective(patchPropertiesDirective string) error {
+	if patchPropertiesDirective != "" {
```

Comment:
Consider just returning early if empty. (It means I can pop the condition from the stack in my head.)